xml/rmt_public_cloud.xml

```diff
@@ -0,0 +1,171 @@
+<variablelist>
+  <varlistentry>
+    <term>Disk space</term>
+    <listitem>
+      <para>
+        The &rmt; server requires sufficient disk space to mirror the repositories.
+        Downloaded packages are available in <filename>/var/lib/rmt/public/repo/</filename>. Disk space is dependent
+        on the number of repositories you mirror. We recommend a minimum of 1.5 times the total size of all enabled repositories.
+        It is a best practice to provision an additional disk volume to support this requirement.
+        You can either mount the volume to <filename>/var/lib/rmt/public/repo/</filename> on instance creation or immediately after the launch.
+        For Azure, it is an additional disk volume. An Amazon EBS volume for AWS and a persistent disk volume for GCP.
+              </para>
+         </listitem>
+  </varlistentry>
+  <varlistentry>
+    <term>IP or DNS resolution</term>
+    <listitem>
+      <para>
+        A static IP address or a DNS name is required in order for clients to connect to the &rmt; server.
+        In Azure, AWS and GCP, a Cloud Service Provider(CSP) provided DNS is assigned when the instance is launched.
+        This IP or DNS may change if the instance is recreated. For Azure, consider using a static IP address to provide a consistent
+        connection point for your clients. In AWS, this would be an elastic IP or Route53. In GCP, this would be a cloud DNS
+        record that uses a static IP address.
+        </para>
+         </listitem>
+  </varlistentry>
+  <varlistentry>
+    <term>Connectivity for the &rmt; server</term>
+    <listitem>
+      <para>
+      The &rmt; server can connect to SCC on ports 80 and 443. There are many ways to provide connectivity.
+For example, in AWS, connectivity to SCC can be provided via an AWS Internet Gateway, an AWS NAT Gateway, or via a local data center (VPN/DX Connection) but this is dependent
+on whether the &rmt; instance is in a public or private subnet. GCP provides direct access to the internet via the VPC routes or
+Cloud NAT services. Azure offers similar services.
+        </para>
+         </listitem>
+  </varlistentry>
+  <varlistentry>
+    <term>Connectivity for clients</term>
+    <listitem>
+      <para>
+        Clients can connect to &rmt; on ports 80 and 443.
+        When launching the &sles; instance to support &rmt; check if network connectivity allows inbound access to the &rmt; server from your clients (HTTP/HTTPS).
+  For AWS, when a &sles; instance is launched to support &rmt;, it is possible to use an existing AWS security group or a new one.
+  The security group must be configured to allow inbound access to the &rmt; server from your clients.
+  For GCP, the firewall rules must be configured to allow inbound access to the &rmt; server from your clients.
+   </para>
+         </listitem>
+  </varlistentry>
+</variablelist>
+</sect1>
+<sect1>
+  <title>More information</title>
+  <variablelist>
```

Comment:
I'd suggest using an `itemizedlist` here, as it is generally a convention to first mention the item of interest and then provide the link to it.

```
<itemizedlist>
      <listitem>
        <para>
          Find organization credentials to create a local mirror of the &sle; repositories
          at <link xlink:href="https://documentation.suse.com/sles/15-SP6/html/SLES-all/cha-rmt-mirroring.html#sec-rmt-mirroring-credentials"/>.
        </para>
      </listitem>
      <listitem>
        <para>
          Register and activate &sles; with the <link xlink:href="https://documentation.suse.com/sles/15-SP6/html/SLES-all/cha-register-sle.html#sec-register-sle-installation/">SUSE Customer Center</link>.
        </para>
      </listitem>
    </itemizedlist>
```

Reply:
Done